src/backend/Shared/ApiResponse/ApiResponse/ActionResults/ControllerExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ApiResponse.ActionResults.NotificationResult;
+using ApiResponse.ActionResults.ZipResult;
+
+using Microsoft.AspNetCore.Mvc;
+
+using System.Collections.Generic;
+
+namespace ApiResponse.ActionResults
+{
+	public static class ControllerExtensions
+	{
+		public static NotificationActionResultBuilder<T> Notify<T>(this ControllerBase controller, T response)
+		{
+			return new NotificationActionResultBuilder<T>(response);
+		}
+
+		public static ZipActionResult Zip(this ControllerBase controller, IEnumerable<FileItem> response, string fileName = "Photos")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=5223686)